src/ch/imedias/rsccfx/view/RsccSupportView.java

```diff
@@ -81,23 +81,23 @@ private void layoutForm() {
     // this.setPadding(new Insets(5, 25, 5, 25)); // TODO: set paddings for "center"
     // this.setId("SupporterView");
 
-    //enterTokenLbl.setFont(new Font(25));
-    enterTokenLbl.setId("EnterTokenLbl");
+    //enterKeyLbl.setFont(new Font(25));
+    enterKeyLbl.setId("EnterKeyLbl");
 
     keyDescriptionLbl.setWrapText(true);
 
-    tokenFld.setFont(new Font(30)); // TODO: Move to CSS
+    keyFld.setFont(new Font(30)); // TODO: Move to CSS
 
     isValidImg.setSmooth(true);
 
-    tokenValidationBox.getChildren().addAll(tokenFld, isValidImg);
-    tokenValidationBox.setSpacing(5);       // TODO: Move to CSS.
-    tokenValidationBox.setHgrow(tokenFld, Priority.ALWAYS);
-    tokenValidationBox.setAlignment(Pos.CENTER_LEFT);
+    keyValidationBox.getChildren().addAll(keyFld, isValidImg);
```

Comment:
Question is: "isValidImg" a good name after all? "validationImg" seems to meet the other namings better. What do you think?